packages/agoric-cli/src/lib/wallet.js

```diff
@@ -37,7 +37,7 @@ export const getLastUpdate = (addr, { readLatestHead }) => {
  * @param {Pick<import('stream').Writable,'write'>} [stdout]
  */
 export const outputAction = (bridgeAction, stdout = process.stdout) => {
-  const capData = marshaller.serialize(bridgeAction);
+  const capData = marshaller.serialize(harden(bridgeAction));
```

Comment:
Though obviously distinct from the point of this PR, as long as we're touching calls to the deprecated `serialize` (and `deserialize`?) can we take the opportunity to change that to `toCapData` (and `fromCapData`?)?

```suggestion
  const capData = marshaller.toCapData(harden(bridgeAction));
```

Reply:
Separately, what changed such that the old code did not need a `harden` here but the new code does? This question also applies to several other occurrences in this PR.

Reply:
The `marshaller` here was not previously a real `makeMarshal()` marshaller, it was the bespoke JSON-plus-replacer thing, that didn't insist upon Remotables or hardened inputs.